internal/rbd/controllerserver.go

```diff
@@ -1086,7 +1099,9 @@ func cloneFromSnapshot(
 	ctx context.Context,
 	rbdVol *rbdVolume,
 	rbdSnap *rbdSnapshot,
-	cr *util.Credentials) (*csi.CreateSnapshotResponse, error) {
+	cr *util.Credentials,
+	parameters map[string]string,
+	updateMetadata bool) (*csi.CreateSnapshotResponse, error) {
```

Comment:
instead of passing the additional bool, just use `nil` or an empty map as `parameters`?